.editorconfig

```diff
@@ -0,0 +1,47 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+ij_any_blank_lines_after_class_header = 0
```

Comment:
The code is not particularly consistent in that regard, however, I guess it would make sense to specify a preferred style